packages/account/src/Styles/account.scss

```diff
@@ -359,133 +473,144 @@ $MIN_HEIGHT_FLOATING: calc(
         @include typeface(--paragraph-left-normal-black, none);
         color: var(--text-general);
     }
-    .account-management__message {
-        margin-top: 1em;
-        font-weight: bold;
-        font-size: var(--text-size-s);
-        color: var(--text-general);
+    .account-management {
+        &__message {
+            margin-top: 1em;
+            font-weight: bold;
+            font-size: var(--text-size-s);
+            color: var(--text-general);
 
-        &-content {
-            display: flex;
-            width: 100%;
-            height: 100%;
-            margin: 0 16px;
-            align-items: center;
-            flex-direction: column;
+            &-content {
+                display: flex;
+                width: 100%;
+                height: 100%;
+                margin: 0 16px;
+                align-items: center;
+                flex-direction: column;
 
-            .dc-btn {
-                height: 4rem;
-                margin-top: 3.2rem;
+                .dc-btn {
+                    height: 4rem;
+                    margin-top: 3.2rem;
+
+                    @include mobile {
+                        margin: 1.6rem 0;
+                        padding: 1.6rem;
+                        width: 100%;
+                    }
+                }
+            }
+            &-icon {
+                width: 12.8rem;
+                height: 12.8rem;
+                margin-top: 8rem;
 
                 @include mobile {
-                    margin: 1.6rem 0;
-                    padding: 1.6rem;
-                    width: 100%;
+                    margin-top: 4rem;
+
+                    /* iPhone SE screen height fixes due to UI space restrictions */
+                    @media only screen and (max-height: 480px) {
+                        margin-top: 3.2rem;
+                    }
                 }
             }
-        }
-        &-icon {
-            width: 128px;
-            height: 128px;
-            margin-top: 80px;
+            &-title {
+                font-size: var(--text-size-xs);
+                margin-bottom: 1em;
```

Comment:
Just checking, do you mean `em` or `rem`?

Reply:
This is not something I've added but I've changed it.